a4c-brooklyn-plugin/pom.xml

```diff
@@ -111,7 +83,22 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-software-database</artifactId>
             <version>${brooklyn.version}</version>
-            <scope>test</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
Could you move these dependencies back to where they were? Trying to keep ordering of `compile` then `test`.